.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+sudo: required
+dist: xenial
+python:
+  - "3.6"
+  - "3.7"
+# command to install dependencies
+install:
+  - pip install -q -r test_requirements.txt
+  - python setup.py -q install
+  - pip freeze
+# command to run tests
+script:
+    - flake8 .
+    - mypy csob_paymentgateway
```

Comment:
měli bychom se naučit používat tox

Reply:
Já tam pak tox mám ale stejně mi přijde lepší ho používat jen na locale a v travisi mít ty verze paralerně